src/plugins/overlap.js

```diff
@@ -137,6 +137,18 @@
                             handleOverlaps(oldRow.tasks);
                         }
                     });
+
+                    api.tasks.on.add(scope, function(task) {
+                        // TODO: Mimicked functionality from api.data.on.change to defer until element creation, but not ideal.  Refactor necessary to raise 'add' event after task is fully drawn.
```

Comment:
There's a viewChange event, but I should implement viewAdd and viewRemove too for this timeout to be removed.